src/common/types/config.ts

```diff
@@ -1,5 +1,25 @@
 import { Server } from './server';
 
+export interface ConfigFile {
```

Comment:
Since there are 2 very similar interfaces, could you please add a comment for each one of them explaining what they are used for and their differences?

Reply:
done.